src/main/java/com/rapid7/client/dcerpc/mssamr/SAMPRDomainPasswordInfo.java

```diff
@@ -0,0 +1,166 @@
+    /**
+     * A 32-bit bit field indicating the password properties policy setting. The defined bits are shown in the
+     * following table. All bits can be combined using a logical OR in any combination. Undefined bits SHOULD be
+     * persisted by the server (that is, stored in its database) and returned to future queries. Clients SHOULD ignore
+     * undefined bits.
+     *
+     * +---------------------------------+--------------------------------------------------------------------------+
+     * | Name/value                      | Description                                                              |
+     * +---------------------------------+--------------------------------------------------------------------------+
+     * | DOMAIN_PASSWORD_COMPLEX         | The server enforces password complexity policy.                          |
+     * | 0x00000001                      |                                                                          |
+     * +---------------------------------+--------------------------------------------------------------------------+
+     * | DOMAIN_PASSWORD_NO_ANON_CHANGE  | Reserved. No effect on password policy.                                  |
+     * | 0x00000002                      |                                                                          |
+     * +---------------------------------+--------------------------------------------------------------------------+
+     * | DOMAIN_PASSWORD_NO_CLEAR_CHANGE | Change-password methods that provide the cleartext password are disabled |
+     * | 0x00000004                      | by the server.                                                           |
+     * +---------------------------------+--------------------------------------------------------------------------+
+     * | DOMAIN_LOCKOUT_ADMINS           | Reserved. No effect on password policy.                                  |
+     * | 0x00000008                      |                                                                          |
+     * +---------------------------------+--------------------------------------------------------------------------+
+     * | DOMAIN_PASSWORD_STORE_CLEARTEXT | The server MUST store the cleartext password, not just the computed      |
+     * | 0x00000010                      | hashes.                                                                  |
+     * +---------------------------------+--------------------------------------------------------------------------+
+     * | DOMAIN_REFUSE_PASSWORD_CHANGE   | Reserved. No effect on password policy.                                  |
+     * | 0x00000020                      |                                                                          |
+     * +---------------------------------+--------------------------------------------------------------------------+
+     */
+    private short minimumPasswordLength;
```

Comment:
Both `MinPasswordLength` `PasswordHistoryLength ` are `unsigned short`, so needs to be represented as `int` in Java.